module/VuFind/src/VuFind/Controller/Plugin/Captcha.php

```diff
@@ -128,11 +128,12 @@ public function verify(): bool
         }
 
         if (!$captchaPassed && $this->errorMode != 'none') {
+            $message = $captcha->getErrorMessage() ?: 'captcha_not_passed';
```

Comment:
Would it make more sense to put the `captcha_not_passed` message in the abstract base class rather than using an empty string there and defaulting it here?

Reply:
Fixed.